src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/ViewModels/WorkflowTrimmingViewModel.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Linq;
+using OrchardCore.Workflows.Models;
+
+namespace OrchardCore.Workflows.Trimming.ViewModels;
+
+public class WorkflowTrimmingViewModel
+{
+    public DateTime? LastRunUtc { get; set; }
+
+    public bool Disabled { get; set; }
+
+    public int RetentionDays { get; set; }
+
```

Comment:
```suggestion

[BindNever]
```

Reply:
`Statuses` needs to be bound. This would break status editing.

Reply:
usually we have two properties, one for the binding and another for the items. In this case, I would create `string Status { get; set; }` property for binding the selected value. Then the items `Statuses` will have `[BindNever]` to ensure we don't bind items.

So I suggest that we stick with the same common approach we follow everywhere else.

Reply:
The status is not a single item but zero to many selections. It doesn't make sense to bind it to a string value.